serverside/zerotrace/example/util.go

```diff
@@ -106,8 +113,8 @@ func isValidUUID(u string) bool {
 	return err == nil
 }
 
-func fmtTimeMs(value time.Duration) float64 {
-	return (float64(value) / float64(time.Millisecond))
+func fmtTimeUs(value time.Duration) float64 {
+	return (float64(value) / float64(time.Microsecond))
```

Comment:
`time.Duration` has a `Microsecond()` method, which we could use here. This should be fine for now though.